server/routes/allocationsMigration/allocationsMigrationController.ts

```diff
@@ -34,8 +34,26 @@ export default class AllocationsMigrationController {
 
     const decoratedMigrations = migrations.map(AllocationsMigrationController.withFilter).map(history => ({
       ...history,
-      applicationInsightsLink: AllocationsMigrationController.applicationInsightsUrl(
-        AllocationsMigrationController.alreadyMigratedApplicationInsightsQuery(history.whenStarted, history.whenEnded),
+      appInsightsAlreadyMigratedLink: AllocationsMigrationController.applicationInsightsUrlNoTimespan(
+        AllocationsMigrationController.alreadyMigratedAppInsightsQuery(
+          history.migrationId,
+          history.whenStarted,
+          history.whenEnded,
+        ),
+      ),
+      appInsightsFailuresLink: AllocationsMigrationController.applicationInsightsUrlNoTimespan(
+        AllocationsMigrationController.failedMigrationAppInsightsQuery(
+          history.migrationId,
+          history.whenStarted,
+          history.whenEnded,
+        ),
+      ),
+      appInsightsFullMigrationLink: AllocationsMigrationController.applicationInsightsUrlNoTimespan(
+        AllocationsMigrationController.fullMigrationAppInsightsQuery(
+          history.migrationId,
+          history.whenStarted,
+          history.whenEnded,
+        ),
```

Comment:
As already done for activities, add 3 App Insights links to the migration list to show already migrated, failures and the all migration events.